src/cli.ts

```diff
@@ -119,29 +90,18 @@ export class CLI {
             CLI.parameters.blobKey = CLI.commander.blobKey;
         }
 
-        if (CLI.commander.threads) {
-            let threads = parseInt(CLI.commander.threads);
-            if (threads) CLI.parameters.threads = threads;
+        if (CLI.commander.processes) {
+            let threads = parseInt(CLI.commander.processes);
+            if (threads) CLI.parameters.processes = threads;
         }
 
         if (CLI.commander.crawl) {
             CLI.parameters.crawlVersion = CLI.commander.crawl;
         }
-
-    }
-
-    /**
-     * Log CLI.parameters, sensitive parameters will be hidden.
-     */
-    public static logParms() {
-        console.log("Runtime parameters:");
```

Comment:
You can enter some environment variables in TravisCI. After saving, they are not visible anymore. You can save an API key there and it will be safe. However, the logs are not protected. That's why I introduced some additional protection. It's not needed if you don't log, but you have to be super careful about this.